.github/workflows/forge-pfn.yaml

```diff
@@ -23,10 +23,6 @@ on:
   pull_request:
     paths:
       - ".github/workflows/forge-pfn.yaml"
-      - "testsuite/find_latest_image.py"
-  push:
-    branches:
-      - aptos-release-v* # The aptos release branches
```

Comment:
should we also stop running all other forge-stable/unstable, etc tests on release branch pushes, and instead have them triggered ad-hoc ?

Reply:
- I think only the stable runs are currently running on the release branch pushes. 😄 None of the other ones should be.
- I agree with you that they should likely all be removed, but not everyone agrees. We'll need to handle that aspect differently.